### ISSUE TITLE
Error with databinding.MiActivityIntroBinding

### ISSUE THREAD

**Issue by OtavioMiguel19** [heinrichreimer/material-intro#261]
I had this erros today. How to solve it?

```
[...]
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.vidroid.com.br.apps.organizze.pro/com.vidroid.com.br.apps.organizze.pro.activity.start.SliderActivity}: java.lang.ClassCastException: com.vidroid.com.br.apps.organizze.pro.databinding.MiActivityIntroBindingImpl cannot be cast to com.heinrichreimersoftware.materialintro.databinding.MiActivityIntroBinding
[...]
Caused by: java.lang.ClassCastException: com.vidroid.com.br.apps.organizze.pro.databinding.MiActivityIntroBindingImpl cannot be cast to com.heinrichreimersoftware.materialintro.databinding.MiActivityIntroBinding
        at com.heinrichreimersoftware.materialintro.app.IntroActivity.onCreate(IntroActivity.java:198)
        at com.vidroid.com.br.apps.organizze.pro.activity.start.SliderActivity.onCreate(SliderActivity.java:31)
[...]
```

I already read the articles [1](https://github.com/heinrichreimer/material-intro/issues/220) and [2](https://github.com/HeinrichReimer/material-intro/issues/117). The solutions didn't work to me.

**Comment by dev4vin** [comment]
I have a similar issue. Could not resolve constraint layout after migrating to androidX. An example binding code is attached below.
```java
@NonNull
    public final android.widget.ImageButton miButtonNext;
    @NonNull
    public final android.support.constraint.ConstraintLayout miFrame;
    @NonNull
    public final com.heinrichreimersoftware.materialintro.view.FadeableViewPager miPager;
```

**Comment by OtavioMiguel19** [comment]
@dev4vin  I _solved_ it using the 1.6 version of Material Intro. Try it.

**Comment by dev4vin** [comment]
I have tried 1.6, unfortunately, gradle could not resolve it. 

```java
implementation 'com.heinrichreimersoftware:material-intro:1.6'
```

**Comment by dev4vin** [comment]
@OtavioMiguel19  It resolved eventually, it was a problem with gradle.
Thanks.

**Comment by abhijitvalluri** [comment]
Thanks @OtavioMiguel19 but this must be fixed soon! Looks like the last release was more than a one and a half years ago!! 😲 Is this project abandoned or is it still actively developed?

**Comment by OtavioMiguel19** [comment]
It is a reeeally good question @abhijitvalluri ! It should at least receive new updates with Android and the new Material Components UI, with bug fixes and new features...

I would like to know if the devs are going to develop more in this project.

**Comment by Mauker1** [comment]
Got the same problem after I've updated AS to 3.2...

**Comment by bewithdhanu** [comment]
I am using 1.6.2 and I am facing the mentioned issue 

> Error with databinding.MiActivityIntroBinding

If I use v1.6, There is no problem
Can you explain?

**Comment by chrisonline** [comment]
Same to me with 1.6.2!

**Comment by chrisonline** [comment]
The same issue is here with some more info: #256

**Comment by adrcotfas** [comment]
#266 should resolve the AndroidX issues.
Until it's merged, I'll be using `implementation 'com.github.adrcotfas:material-intro:e1228'` which points to the last commit on my branch.

**Comment by VladimirWrites** [comment]
I had the same issue, and I just manage to solve it. https://stackoverflow.com/questions/55024316/java-lang-classcastexception-for-include-tag-using-databinding/
In your case, `IntroActivity` needs to be defined in `library/AndroidManifest.xml` file.

**Comment by matheuslipk** [comment]
Resolvi meu problema com a versão 1.6

**Comment by chrisonline** [comment]
@matheuslipk Can you please write in english as everyone here?

**Comment by bewithdhanu** [comment]
> Resolvi meu problema com a versão 1.6

@matheuslipk says, **I solved my problem with version 1.6**

**Comment by TangLiuRain** [comment]
> @dev4vin I _solved_ it using the 1.6 version of Material Intro. Try it.

useful！

**Comment by chimbori** [comment]
This should now be resolved with Release 2.0.0.

**Comment by abhijitvalluri** [comment]
Yay!!
